### PR TITLE
Small updates to ServerManagement API 

### PR DESCRIPTION
**Pull request by fearthecowboy**

### CODE REVIEW

File: arm-servermanagement/2015-07-01-preview/servermanagement.json [hunk 1]
```diff
@@ -1633,12 +1607,40 @@
...
+        "gatewayVersion": {
+          "type": "string",
+          "description": "The version of the gateway that is installed on the system."
+        },
+        "friendlyOsName": {
+          "type": "string",
+          "description": "The Plaintext description of the OS on the gateway."
+        },
+        "installedDate": {
+          "type": "string",
```
Comment (amarzavery) [inline arm-servermanagement/2015-07-01-preview/servermanagement.json]: missing description

File: arm-servermanagement/2015-07-01-preview/servermanagement.json [hunk 2]
```diff
@@ -1633,12 +1607,40 @@
...
+          "type": "integer",
+          "description": "Number of logical processors in the gateway system."
+        },
+        "name": {
+          "type": "string",
+          "description": "The computer name of the gateway system."
+        },
+        "gatewayId": {
+          "type": "string",
+          "description": "the gateway resource id."
```
Comment (amarzavery) [inline arm-servermanagement/2015-07-01-preview/servermanagement.json]: **T**he gateway ...

File: arm-servermanagement/2015-07-01-preview/servermanagement.json [hunk 3]
```diff
@@ -1633,12 +1607,40 @@
           "type": "number",
           "description": "The available memory on the gateway host machine in megabytes."
         },
-        "gatewayCpuPercent": {
+        "gatewayCpuUtilizationPercent": {
           "type": "number",
```
Comment (amarzavery) [inline arm-servermanagement/2015-07-01-preview/servermanagement.json]: Do you expect percentage in float/decimal ? If so then add `"format": "decimal" / "float"` as applicable.
If you expect the percentage number to be integers then this should be `"type": "integer", "format": "int32"`

File: arm-servermanagement/2015-07-01-preview/servermanagement.json (hunk 1 above)
Comment (fearthecowboy) [inline arm-servermanagement/2015-07-01-preview/servermanagement.json]: fixed. (thanks!)

File: arm-servermanagement/2015-07-01-preview/servermanagement.json (hunk 2 above)
Comment (fearthecowboy) [inline arm-servermanagement/2015-07-01-preview/servermanagement.json]: Fixed

File: arm-servermanagement/2015-07-01-preview/servermanagement.json (hunk 3 above)
Comment (fearthecowboy) [inline arm-servermanagement/2015-07-01-preview/servermanagement.json]: Fixed (thanks! I was unaware of `format=float`; )

**Comment by amarzavery** [comment]
welcome :). Shall merge the PR once the CI is green.

**Pull request merged**